### PR TITLE
Documentation Formatting

### PR DESCRIPTION
**Pull request by SkezzaB**
### Summary

Documentation formatting

### Checklist

- [ ] If code changes were made then they have been tested.
    - [ ] I have updated the documentation to reflect the changes.
- [x] This PR fixes an issue.
- [ ] This PR adds something new (e.g. new method or parameters).
- [ ] This PR is a breaking change (e.g. methods or parameters removed/renamed)
- [x] This PR is **not** a code change (e.g. documentation, README, ...)

### CODE REVIEW

File: discord/iterators.py [hunk 1]
```diff
@@ -182,7 +182,7 @@ class HistoryIterator(_AsyncIterator):
     `after` parameter to the newest message received. If messages are not
     reversed, they will be out of order (99-0, 199-100, so on)
 
-    A note that if both before and after are specified, before is ignored by the
+    A note that if both `before` and `after` are specified, `before` is ignored by the
```
Comment (Gobot1234) [inline discord/iterators.py:185]: This should be double base ticked right?

File: discord/iterators.py [hunk 2]
```diff
@@ -471,7 +471,7 @@ class GuildIterator(_AsyncIterator):
     parameter to the newest guild received, If guilds are not reversed, they
     will be out of order (99-0, 199-100, so on)
 
-    Not that if both before and after are specified, before is ignored by the
+    Not that if both `before` and `after` are specified, `before` is ignored by the
```
Comment (Gobot1234) [inline discord/iterators.py:474]: Same with this

File: discord/iterators.py (hunk 1 above)
Comment (SkezzaB) [reply discord/iterators.py]: Fixed in the latest commit

File: discord/iterators.py (hunk 2 above)
Comment (SkezzaB) [reply discord/iterators.py]: This too

**Pull request closed without merging**